build/wrapper.cc

```diff
@@ -257,62 +257,68 @@ JuliaGcsClient::JuliaGcsClient(const std::string &gcs_address) {
 }
 
 Status JuliaGcsClient::Connect() {
-    gcs_client_ = std::make_unique<gcs::PythonGcsClient>(options_);
-    return gcs_client_->Connect();
+    io_service_ = std::make_unique<instrumented_io_context>();
+    io_service_thread_ = std::make_unique<std::thread>([this] {
+        std::unique_ptr<boost::asio::io_service::work> work(
+            new boost::asio::io_service::work(*io_service_));
+        io_service_->run();
+    });
+    gcs_client_ = std::make_unique<gcs::GcsClient>(options_);
+    return gcs_client_->Connect(*io_service_);
+}
+
+void JuliaGcsClient::Disconnect() {
+    io_service_->stop();
+    io_service_thread_->join();
+    gcs_client_->Disconnect();
+    gcs_client_.reset();
 }
 
-std::string JuliaGcsClient::Get(const std::string &ns,
-                                const std::string &key,
-                                int64_t timeout_ms) {
+std::string JuliaGcsClient::Get(const std::string &ns, const std::string &key) {
     if (!gcs_client_) {
         throw std::runtime_error("GCS client not initialized; did you forget to Connect?");
     }
     std::string value;
-    Status status = gcs_client_->InternalKVGet(ns, key, timeout_ms, value);
+    Status status = gcs_client_->InternalKV().Get(ns, key, value);
     if (!status.ok()) {
         throw std::runtime_error(status.ToString());
     }
     return value;
 }
 
-int JuliaGcsClient::Put(const std::string &ns,
```

Comment:
I kept the original signature for these rather than return `Status` as it made for less changes to the code